framework_tvb/tvb/adapters/simulator/model_forms.py

```diff
@@ -141,7 +141,9 @@ def __init__(self, are_voi_disabled=False):
 
     @staticmethod
     def get_params_configurable_in_phase_plane():
-        return ['tau', 'a', 'b', 'c', 'I', 'd', 'e', 'f', 'g', 'alpha', 'beta', 'gamma']
+        return {"tau": r":math:`\tau`", "a": ":math:`a`", "b": ":math:`b`", "c": ":math:`c`", "I": ":math:`I`",
```

Comment:
Are the values of these dictionaries used? I see the mathjax labels are being taken directly from attributes and not from these dictionaries.